composer.json

```diff
@@ -3,27 +3,58 @@
 	"description": "Forum extension",
 	"license": "GPL-2",
 	"type": "typo3-cms-extension",
-	"autoload": {
-		"psr-4": {
-			"Mittwald\\Typo3Forum\\": "Classes"
-		}
-	},
+	"version": "1.0.3",
```

Comment:
The master version of the `composer.json` should not contain a version number. If necessary, we can add a build script  (like at [web2pdf](https://github.com/mittwald/typo3-web2pdf/blob/master/build-release.sh)) to add the version number when a new release is built.